downloaded/lisk_bridge.sh

```diff
@@ -21,45 +21,46 @@
 ######################################################################
 
 # Declare working variables
-TARGET_HEIGHT="3513100"
 BRIDGE_HOME="$(pwd)"
 BRIDGE_NETWORK="main"
-LISK_HOME="$HOME/lisk-main"
 
 # Reads in required variables if configured by the user.
 parseOption() {
 	OPTIND=1
-	while getopts ":s:b:n:h:" OPT; do
+	while getopts ":s:n:h:" OPT; do
 		# shellcheck disable=SC2220
 		case "$OPT" in
-			 s) LISK_HOME="$OPTARG" ;;
-			 b) BRIDGE_HOME="$OPTARG" ;; # Where the bridge is located
-			 n) BRIDGE_NETWORK="$OPTARG" ;; # Which network is being bridged
-			 h) TARGET_HEIGHT="$OPTARG" ;; # What height to cut over at
-		 esac
-	 done
+			s) LISK_HOME="$OPTARG" ;;
+			n) BRIDGE_NETWORK="$OPTARG" ;; # Which network is being bridged
+			h) TARGET_HEIGHT="$OPTARG" ;; # What height to cut over at
+			:) echo 'Missing argument for -'"$OPTARG" >&2;
+			   SHOW_USAGE=1;;
+			*) echo 'Unimplemnted option: -'"$OPTARG">&2;
+			   SHOW_USAGE=1 ;;
+		esac
+	done
+	if [[ $SHOW_USAGE ]]; then
+		usage
+		exit 1;
+	fi
+	if [[ ! $LISK_HOME ]]; then
+		LISK_HOME="$HOME/lisk-$BRIDGE_NETWORK"
+	fi
+	JQ="$LISK_HOME/bin/jq"
 }
 
 # Harvests the configuation data from the source installation
 # for an automated cutover.
 extractConfig() {
-	PM2_CONFIG="$LISK_HOME/etc/pm2-lisk.json"
-	LISK_CONFIG="$(grep "config" "$PM2_CONFIG" | cut -d'"' -f4 | cut -d' ' -f2)" >> /dev/null
-	LISK_CONFIG="$LISK_HOME/$LISK_CONFIG"
+	LISK_CONFIG="$LISK_HOME/config.json"
 	export PORT
```

Comment:
Why create a function (that gets called only once) if all is does is `export` one variable?